doc/source/whatsnew/v0.16.0.txt

```diff
@@ -190,6 +190,15 @@ Enhancements
 - Added ``StringMethods.find()`` and ``rfind()`` which behave as the same as standard ``str`` (:issue:`9386`)
 
 - Added ``StringMethods.isnumeric`` and ``isdecimal`` which behave as the same as standard ``str`` (:issue:`9439`)
+- The ``read_excel()`` function's :ref:`sheetname <_io.specifying_sheets>` argument now accepts a list and None, to get 
```

Comment:
back-ticks on None, don't put spaces the end of a statement, just continue it on (it will get wrapped automatically)